lib/Conversion/TorchToLinalg/PopulatePatterns.h

```diff
@@ -45,6 +45,9 @@ void populateLinearPatternsAndLegality(TypeConverter &typeConverter,
 void populatePoolingPatternsAndLegality(TypeConverter &typeConverter,
                                         RewritePatternSet &patterns,
                                         ConversionTarget &target);
+void populateAdaptivePoolingPatternsAndLegality(TypeConverter &typeConverter,
+                                        RewritePatternSet &patterns,
```

Comment:
nit: indentation is off. Make sure to format all your changes with `git clang-format`